test/testSXSModes.py

```diff
@@ -0,0 +1,373 @@
+""" Test SXS waveform load against waveformtools.
+
+It was noted that the RAW modes data loaded from H5 files
+were exactly equal. Howeverm after resampling to an axis,
+there can be differences due to a differencein the 
+time axis and interpolation. This script demonstrates the 
+level of precision that can be expected from comparison
+of waveforms that requires interpolation and resampling.
+"""
+
+import os
+import sys
+
+import h5py
+import numpy as np
+
+cwd = os.getcwd()
+from pathlib import Path
+
+home = Path.home()
+
+libpath = f"{cwd}/../"
+
+if libpath not in sys.path:
+    sys.path.append(libpath)
+
+import unittest
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from nrcatalogtools.sxs import SXSCatalog
+# from pycbc.waveform.utils import coalign_waveforms
+from pycbc import pnutils
+from pycbc.filter.matchedfilter import match
+# pycbc
+# from pycbc.waveform import td_approximants
+from pycbc.types.timeseries import TimeSeries
+# waveformtools
+from waveformtools.waveforms import modes_array
```

Comment:
`waveformtools` should be added as a dependency to `requirements.txt` and `environment.yaml`

Reply:
I haven't published my repo to pypi. Do you suggest I do it?

Reply:
If its public already, then why not?